gpii/configs/fm.ps.sr.dr.mm.os.lms.production.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
I'm renaming these top-level configs in line with the naming conventions lower down - given each one corresponds to a GRADE, they will all go into the "gpii.config" namespace. To avoid confusion with the other configs which just define a flowmanager, I'll put this one into gpii.config.trusted-flowmanager.production - what do you think?